packages/dd-trace/src/config.js

```diff
@@ -201,16 +201,22 @@ class Config {
       false
     )
 
-    let appsec = options.appsec || (options.experimental && options.experimental.appsec)
+    let appsec = options.appsec != null ? options.appsec : options.experimental && options.experimental.appsec
```

Comment:
Given the loose equality operator, isn't this the same as before?

Reply:
Same below.

Reply:
the loose operator is to check if it's null or undefined, to be backward compatible with the experimental option, it had nothing to do with the isTrueOrFalse thing from before we talked about ?

Reply:
It's different from the previous version because if the value of `options.appsec` is false, I want to use that false value. I only bypass it if it's null or undefined

Reply:
Ah yeah gotcha. I'm not a fan of loose equality generally speaking and would have preferred a check for a boolean, but  not gonna block on this.